ProcessManager.php

```diff
@@ -723,7 +723,7 @@ protected function bootstrapFailed(Connection $conn)
 
             foreach ($this->slaves as &$slave) {
                 $slave['keepClosed'] = true;
-                if(true === isset($slave['connection']) && true === is_object($slave['connection'])) {
+                if(true == isset($slave['connection']) && true == is_object($slave['connection'])) {
```

Comment:
@emgiezet I don't see why you need this check. Could you please describe a case, where  `$slave['connection']` can be neither `null` nor the `Connection` instance?